tests/update_reference_data.sh

```diff
@@ -102,7 +102,7 @@ for test_name in ${tests}; do
   then
     copyToReferenceDir \
       $configuration/build-opm-simulators/tests/results/flow+ctaquifer_2d_oilwater/ \
-      $OPM_TESTS_ROOT/aquifer-oilwater/opm-simulation-reference/flow \
+      $OPM_TESTS_ROOT/aquifer-oilwater/eclipse-simulation \
```

Comment:
Sorry, but I am not sure this makes sense. @akva2 , any comment?

Reply:
Integration tests testing against the reference result under `opm-simulation-reference`, we need to make sure the `opm-simulation-reference` always agrees well with the `eclipse-simulation` result by ourselves, while we do not compare the results against the `eclipse-simulation` during integration tests. This is my understanding.